catroid/src/org/catrobat/catroid/ui/dialogs/OrientationDialog.java

```diff
@@ -143,4 +125,53 @@ public void setCreateEmptyProject(boolean isChecked) {
 	public void setCreateDroneProject(boolean isChecked) {
 		createDroneProject = isChecked;
 	}
+
+	private class LoadingOnCreatingProject extends AsyncTask<String, Void, Boolean> {
+		private ProgressDialog dialog;
+
+		protected LoadingOnCreatingProject(Context context) {
+			dialog = new ProgressDialog(context);
+		}
+
+		@Override
+		protected void onPostExecute(Boolean showing) {
+			super.onPostExecute(showing);
+			dialog.dismiss();
+		}
+
+		@Override
+		protected void onPreExecute() {
+			super.onPreExecute();
+			this.dialog.setMessage(getResources().getString(R.string.please_wait));
```

Comment:
Remove super call, create ProgressDialog here (Instead of context, use `getActivity()`). Also remove `this.` here and the next line.